doc/user-guide/Jingle-SIP-setup.md

```diff
@@ -0,0 +1,300 @@
+## Jingle/SIP setup proof of concept
+
+This tutorial will show you how to configure MongooseIM, Routr (a SIP server) and client applications to demonstrate how the Jingle/SIP integration works.
+
+### Prerequisites
+
+We are going to use the following open source software:
+
+* MongooseIM - https://github.com/esl/MongooseIM
+    * see [How-to-build](How-to-build.md) for details on building.
+      It's important to remember to run the configuration script with `with-jingle-sip` flag set:
+      `tools/configure with-jingle-sip`.
+      Without this, third party dependencies required by the Jingle/SIP translator will not be included in the release.
+
+* Routr (SIP server) - https://routr.io
```

Comment:
Can we use special syntax for adding links? So that it will redirect to the page after clicking.
its already done for `How-to-build` link, above